### ISSUE TITLE
Where is /style/mergely-theme/jquery-ui-1.10.1.custom.css accessed from editor/editor.php ?

### ISSUE THREAD

**Issue by mnemonicflow** [wickedest/Mergely#59]
`
editor/editor.php
25:     <link type="text/css" rel="stylesheet" href="/style/mergely-theme/jquery-ui-1.10.1.custom.css" />
`
I see a 404 response , due to missing directory and file. Is the mergely-theme no longer available?

**Comment by wickedest** [comment]
Works [for me](http://www.mergely.com/style/mergely-theme/jquery-ui-1.10.1.custom.css).  If you're trying to load my editor and expect it to work, then it's not really supported.  Have a look at the examples.

**Comment by mnemonicflow** [comment]
I'm trying to use the code from this repo as is, on my own server.

**Comment by wickedest** [comment]
As I said, trying to use editor.php directly will not work.  As you discovered, it's themed.  It has custom ajax API handlers.  It has my own menu.  These things were never available and never supported.  You need to make your own integration editor and theme as per the requirements for your application.

**Comment by mnemonicflow** [comment]
I get it now. I will have to redo the handling of the AJAX calls on the server side because I deleted the previous document root, thinking that all I need is already provided in this repo. Forgot the details since last time I looked at it...

Although I would really love to have a solution that works out of the box without me having to do anything special beyond basic configuration, so that I can install my own fully functioning web diff tool similar to http://mergely.com/editor on my own servers.

Something like: https://github.com/electerious/Lychee/blob/master/docs/Installation.md#1-requirements && https://github.com/claudehohl/Stikked#installation
